dockstore/dockstore-ui2
# Remove ngx-lite/json-ld dependency

**Description**
We're removing the dependency because they haven't issued newer releases to keep up with new Angular releases.

Copy the relevant part of their code and commit it to our repo, with attribution.

In testing it, I noticed that the json-ld was not showing up on the
home page. I thought I'd broken it, but it turns out that component
is not ever used (json-ld is not showing up in prod either, without my
changes). I deleted the component and moved the json-ld to the
actual home page component.

**Review Instructions**
1. Go to the home page, not logged into Dockstore
2. Open Browser developer tools
3. Verify there are 2 instances of a script tag, with the attribute `type=application/ld+json`

**Issue**
dockstore/dockstore#5218

Please make sure that you've checked the following before submitting your pull request. Thanks!
